db/migrate/20151435234635_add_container_definition_security_contexts.rb

```diff
@@ -0,0 +1,13 @@
+class AddContainerDefinitionSecurityContexts < ActiveRecord::Migration
+  def change
+    add_column    :container_definitions, :privileged, :boolean
+    add_column    :container_definitions, :run_as_user, :bigint
+    add_column    :container_definitions, :se_linux_user, :string
+    add_column    :container_definitions, :se_linux_role, :string
+    add_column    :container_definitions, :se_linux_type, :string
+    add_column    :container_definitions, :se_linux_level, :string
```

Comment:
If you have in mind to re-use the selinux attributes for something else than containers then move it out to another (polymorphic) generic table `security_context` (vs the specific `container_security_context` initially proposed), anyway `privileged`, `run_as_user` and `run_as_non_root` don't seem to belong there.